thorlcr/activities/choosesets/thchoosesetsslave.cpp

```diff
@@ -30,7 +30,7 @@ class BaseChooseSetsActivity : public CSlaveActivity
     unsigned *tallies;
 
 public:
-    BaseChooseSetsActivity(CGraphElementBase *_container) : CSlaveActivity(_container)
+    BaseChooseSetsActivity(CGraphElementBase *_container) : CSlaveActivity(_container, basicActivityStatistics, "traceChooseSets")
```

Comment:
See other comment in CActivityBase ctor.

Suggestion: the flag is set in the container object (CGraphElementBase).
The option name could be based on getActivityText (by default), or a space stripped version of it perhaps, it's probably close enough to what we want in most cases, but could have a separate helper for exceptions.